### DISCUSSION TITLE
How can implement a frontend for resources ?

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [filamentphp/filament#1996] [Help] [answered]
Hi, 

it's possible to publish some resources as list read only mode without login to admin panel ? I can change middleware and routes in some ways or i must create a frontend page ?

Thanks !

Luca

**Comment by danharrin** [comment, marked as answer]
You could use the [table builder](https://filamentphp.com/docs/tables) to build tables with exactly the same syntax outside the admin panel.

**Reply by lucacalcaterra** [reply]
Yeah, already done it, i would like to know if there are possibilities to recycle admin panel directly but this solution seems to be ok. Thanks !
